packages/jazz-tools/package.json

```diff
@@ -164,6 +164,7 @@
     "zod": "3.25.28"
   },
   "scripts": {
+    "check": "tsc --noEmit",
```

Comment:
I found it useful to have this command to quickly validate type errors. If folks don't like it, I can remove it